src/client/lazy-app/Modal/index.tsx

```diff
@@ -0,0 +1,136 @@
+import { h, Component, VNode, Fragment } from 'preact';
+import * as style from './style.css';
+import 'add-css:./style.css';
+import { linkRef } from 'shared/prerendered-app/util';
+import { cleanSet } from '../util/clean-modify';
+import { animateTo } from '../util';
+
+interface Props {
+  icon: VNode;
+  title: string;
+  content: VNode;
+}
+
+interface State {
+  shown: boolean;
+}
+
+export default class Modal extends Component<Props, State> {
+  private dialogElement?: HTMLDialogElement;
+
+  componentDidMount() {
+    if (!this.dialogElement) throw new Error('Modal missing');
+
+    // Set inert by default
+    this.dialogElement.inert = true;
```

Comment:
Can you talk me through the usage of `inert` in this component?

Reply:
I have a feeling it's another instance of obsolete code now that we're using the native `dialog` element, but here's the logic: when hidden, the dialog is made inert to prevent its contents from being `TAB`bed to. Might be a remnant of the time when I didn't use the dialog element and had to ensure the modal wasn't focusable.

Reply:
I think so. `<dialog>` content isn't tabbable when it's not-shown.